### ISSUE TITLE
Align Skip link

### ISSUE THREAD

**Issue by chris-gds** [alphagov/govuk_publishing_components#5047]
## What?

As part of [this PR discussion](https://github.com/alphagov/frontend/pull/2774#discussion_r643120176) it was noticed that noticed that the "Skip Link" anchor link is not as consistent as it could be across `frontend` (and perhaps other repos)

### GOV.UK use:

https://github.com/alphagov/govuk_publishing_components/blob/3b15e3c69698e2f18d95e9e59e2f998dad537779/app/views/govuk_publishing_components/components/_layout_for_public.html.erb#L66  uses `#content`

https://components.publishing.service.gov.uk/component-guide/skip_link uses `#main-content`

### Design System use

https://design-system.service.gov.uk/styles/page-template/ uses `#main-content`

https://design-system.service.gov.uk/components/skip-link/  uses `#content`

**Comment by andysellick** [comment]
@jon-kirwan I'm minded to close this issue, given that all of our applications now use the public layout component, which contains [the main skip link used on all GOV.UK pages](https://github.com/alphagov/govuk_publishing_components/blob/main/app/views/govuk_publishing_components/components/_layout_for_public.html.erb#L76-L78) - so we're at least internally consistent.

I don't think there's any need for us to link to exactly the same anchor as the Design System and the work to change that feels unnecessary. Any objection?

**Comment by jon-kirwan** [comment]
> I don't think there's any need for us to link to exactly the same anchor as the Design System and the work to change that feels unnecessary. Any objection?

No, I’m happy to close it. As you say, the anchor is consistent across the frontend apps, so I agree it’s probably not necessary to change anything.
